README.md

```diff
@@ -1,13 +1,10 @@
 # EHR FHIR Server
 This subproject hosts a HAPI FHIR server that is based on the [hapi-fhir-jpaserver-example](https://github.com/jamesagnew/hapi-fhir/tree/master/hapi-fhir-jpaserver-example).
 
-## Init the test-ehr 
-1. delete `target` and `build` folders from test-ehr if they exist
```

Comment:
Do you think it would be good to still have the cleanup line somewhere else in the README?